### PR TITLE
Node 16.20.2 LTS updated

### PR DESCRIPTION
**Pull request by juanky201271**
Nothing crazy...

### CODE REVIEW

File: ios/ZingoMobile.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -283,7 +283,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "set -e\n\nexport NODE_BINARY=/Users/juancarloscarmonacalvo/.nvm/versions/node/v16.16.0/bin/node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "set -e\n\nexport NODE_BINARY=/Users/juancarloscarmonacalvo/.nvm/versions/node/v16.20.2/bin/node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
```
Comment (Oscar-Pepper) [inline ios/ZingoMobile.xcodeproj/project.pbxproj]: what is this for? why does it specify a path on your local machine? does that mean that this only works on your computer?

File: ios/.xcode.env [hunk 2]
```diff
@@ -1 +1 @@
-NODE_BINARY=/Users/juancarloscarmonacalvo/.nvm/versions/node/v16.16.0/bin/node
+NODE_BINARY=/Users/juancarloscarmonacalvo/.nvm/versions/node/v16.20.2/bin/node
```
Comment (Oscar-Pepper) [inline ios/.xcode.env]: see my other comment

File: ios/ZingoMobile.xcodeproj/project.pbxproj (hunk 1 above)
Comment (juanky201271) [reply ios/ZingoMobile.xcodeproj/project.pbxproj]: You are right... mmmm, I'm going to try to change this in Macos, like `~/`... Thanks.

Comment (juanky201271) [reply ios/ZingoMobile.xcodeproj/project.pbxproj]: done!

File: ios/.xcode.env (hunk 2 above)
Comment (juanky201271) [reply ios/.xcode.env]: done!

**Review by Oscar-Pepper — APPROVED** [review]

**Pull request merged**